src/olympia/reviewers/views.py

```diff
@@ -1362,6 +1363,18 @@ def list(self, request, *args, **kwargs):
         return Response(serializer.data)
 
     def retrieve(self, request, *args, **kwargs):
+        if self.request.GET.get('file_only', False):
+            version = self.get_object()
```

Comment:
nit: we need to do this regardless, so `version = self.get_object()` could be outside of the `if` and used below for `AddonBrowseVersionSerializer` too.